### PR TITLE
fix test register often fail error

### PR DESCRIPTION
**Pull request by victorwangstone**

### CODE REVIEW

**Comment by skylarCai** [comment]
Security Check : 7e992ef947e412e77700af663f9b0fd492a3439f

**Comment by skylarCai** [comment]
auto build success,number:217

**Comment by skylarCai** [comment]
autotest fail,number:1576

**Review by wuxn9999 — APPROVED** [review]
Looks good to me.

File: components/linksdk/core/utils/core_http.c [hunk 1]
```diff
@@ -447,10 +447,6 @@ static int32_t _core_http_recv_header(core_http_handle_t *http_handle, uint32_t
         if (timenow_ms > http_handle->sysdep->core_sysdep_time()) {
             timenow_ms = http_handle->sysdep->core_sysdep_time();
         }
```
Comment (skylarCai) [inline components/linksdk/core/utils/core_http.c:449]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**